ci/goreleaser/goreleaser-el7.yml

```diff
@@ -37,26 +28,17 @@ builds:
       - linux
     goarch:
       - arm64
-  # static builds strip symbols and do not allow plugins
-  - id: static-amd64
-    ldflags:
-      - -s -w -X github.com/TykTechnologies/tyk/gateway.VERSION={{.Version}} -X github.com/TykTechnologies/tyk/gateway.Commit={{.FullCommit}} -X github.com/TykTechnologies/tyk/gateway.buildDate={{.Date}} -X github.com/TykTechnologies/tyk/gateway.builtBy=goreleaser
-      - -linkmode external -extldflags -static
-    goos:
-      - linux
-    goarch:
-      - amd64
-
+    binary: tyk
```

Comment:
This is probablly coming from somewhere else than TD-918, can you provide more context (separate jira ticket id or reasoning for removal?)

Reply:
This is to do with the removal of slim images. The slim distroless images were the only release asset that used this statically linked binary as base. It's almost useless otherwise since static lnking doesn't support plugins. So with the removal of slim images, keeping the static binaries doesn't make sense.